src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```
@@ -283,7 +281,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.22.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.834.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.835.1" />
```

Per Teams discussion, this should now go back to the normal versioning. 4.35.3 supports all needed targets.
```suggestion
    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.35.3" />
```